runtime/drivers/repo.go

```diff
@@ -39,3 +40,8 @@ type RepoObjectStat struct {
 }
 
 var ErrFileAlreadyExists = errors.New("file already exists")
+
+type FileEntry struct {
+	Path  string
+	IsDir bool
+}
```

Comment:
Should we call it `DirEntry` to align with Go's type name for it?

Reply:
Ya makes sense